api/app/src/main/kotlin/packit/service/BasePacketService.kt

```diff
@@ -16,4 +18,25 @@ class BasePacketService(private val packetRepository: PacketRepository) : Packet
     {
         return packetRepository.findAll()
     }
+
+    override fun getChecksum(): String
+    {
+        return packetRepository.findAllIds()
```

Comment:
Are we confident that we'll be assembling our ids in the same order as ourpack when calculating the two checksums?

Reply:
good point, no! this should be sorted.